java/runtime/src/main/java/io/ray/runtime/object/ObjectSerializer.java

```diff
@@ -121,6 +124,30 @@ public static Object deserialize(
     }
   }
 
+  public static NativeRayObject serializeArrow(Object object) {
```

Comment:
Seems that arrow data is already serialized to byte array or byte buffer already. This will have a copy right?  If we want to avoid this copy, we may need `ArrowRecordBatch` here?

Reply:
Yes, this will have a copy. But it's difficult to avoid it. ArrowStreamWriter takes a stream/channel to write to, so we have to wrap the buffer allocated in JNI as a stream. Besides, java arrow does not has MockOutputStream like python to calculate the size. We decide to skip it in this PR